src/points.jl

```diff
@@ -13,8 +13,9 @@ immutable LLA{T <: Number}
     lon::T
     alt::T
 end
+LLA(lat, lon, alt=zero(lat)) = LLA(promote(lat, lon, alt)...)
 LLA(;lat=NaN,lon=NaN,alt=0.0) = LLA(lat,lon,alt) # Constructor that is idependent of storage order
-LLA{T}(lat :: T, lon :: T) = LLA(lat, lon, zero(T))
```

Comment:
Looking at #28 again I should maybe add promotion constructors for the other types. What is your preference on removing this constructor?